tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -460,4 +460,34 @@ internal static class Adfs2019LabConstants
         public const string ADFS2019ClientSecretURL = "https://buildautomation.vault.azure.net/secrets/ADFS2019ClientCredSecret/";
         public const string ADFS2019ClientSecretName = "ADFS2019ClientCredSecret";
     }
+
+    public enum MsiAzureResource
+    {
+        WebApp,
+        Function,
+        Vm,
+        AzureArc,
+        CloudShell,
+        ServiceFabric
+    }
+
+    public enum SendMSIHeader

```

Comment:
It's not clear what these enums do, please add comments.

Reply:
added